integrations-and-supported-tools/pytorch-lightning/notebooks/Neptune_PyTorch_Lightning.ipynb

```diff
@@ -58,6 +58,16 @@
     "! pip install -U neptune-client pytorch-lightning scikit-plot torchvision"
    ]
   },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**Note**: If running on Google Colab, please restart the kernel and continue execution from the next cell to avoid a `ContextualVersionConflict` error.\n",
```

Comment:
```suggestion
    "**Note**: If running on Google Colab, restart the kernel and continue execution from the next cell to avoid a `ContextualVersionConflict` error.\n",
```